src/rating/rating.ts

```diff
@@ -28,6 +40,7 @@ export interface StarTemplateContext {
 @Component({
   selector: 'ngb-rating',
   changeDetection: ChangeDetectionStrategy.OnPush,
+  host: {'(keydown)': 'handleKeyDown($event)'},
```

Comment:
I feel keyup makes more sense. I feel for a rating, I would want to press the button x amount of times for the rating I'm giving.

Holding down seems like it would be harder to select the exact number and would cause more mistakes than precise selections.